frontend/src/services/httpClient.ts

```diff
@@ -12,4 +15,12 @@ const handleErrors = (error: any) => {
 
 httpClient.interceptors.response.use(config => config, handleErrors);
 
+httpClient.interceptors.request.use(config => {
+    const token = loggedUser.token;
```

Comment:
```suggestion
const token = store.getState().loggedUserState.token;
```
I think it will work